src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/AncmIISExpressV2.wixproj

```diff
@@ -7,7 +7,7 @@
     <IsShipping>true</IsShipping>
     <ProjectGuid>17c76489-4c09-4e14-b81c-7a86cd937144</ProjectGuid>
     <OutputType>Package</OutputType>
-    <OutputName>ancm_iis_express_$(Platform)_en_v2</OutputName>
+    <OutputName>$(Name)_$(Platform)</OutputName>
```

Comment:
@jkotalik please double-check the `*.wixproj` changes in this PR. @mmitche described the reasoning in https://github.com/dotnet/aspnetcore/pull/26506#issuecomment-705776687 but I'd appreciate another set of 👀

Reply:
Yes please. I believe this should be fine though.

Reply:
@jkotalik Any concerns here?

Reply:
I think it's okay, but @shirhatti and @joeloff to confirm. Pretty much, is it okay to change the name of the MSIs for IISExpress.

I think the next time we update VS, we will need to react to these name changes. besides that I think it's okay.